src/sentry/migrations/0171_backfill_cool.py

```diff
@@ -0,0 +1,39 @@
+# Generated by Django 1.11.29 on 2021-03-11 06:29
+
+from django.db import migrations
+
+
+def backfill_is_cool(apps, schema_editor):
+    User = apps.get_model("sentry", "User")
+
+    for user in User.objects.all():
```

Comment:
This data migration involved a large table, so its better if we iterate over the entire table instead
```suggestion
    for user in RangeQuerySetWrapperWithProgressBar(User.objects.all()):
```